crates/sui-core/src/checkpoints/mod.rs

```diff
@@ -726,15 +720,19 @@ impl CheckpointBuilder {
                     .await?;
 
                 let committee = system_state_obj.get_current_epoch_committee().committee;
+
+                self.accumulator.accumulate_checkpoint(
```

Comment:
maybe add a comment explaining why this had to be moved (so that it can see the end of epoch tx)